packages/aragon-wrapper/src/index.test.js

```diff
@@ -3,22 +3,57 @@ import sinon from 'sinon'
 import proxyquire from 'proxyquire'
 import { Observable } from 'rxjs/Rx'
 
-const messengerConstructorStub = sinon.stub()
-const utilsStub = {
-  makeAddressMapProxy: sinon.fake.returns({}),
-  makeProxy: sinon.stub(),
-  addressesEqual: Object.is
-}
-const Aragon = proxyquire.noCallThru().load('./index', {
-  '@aragon/messenger': messengerConstructorStub,
-  './utils': utilsStub
-}).default
+test.beforeEach(t => {
+  const messengerConstructorStub = sinon.stub()
+  const utilsStub = {
+    makeAddressMapProxy: sinon.fake.returns({}),
+    makeProxy: sinon.stub(),
+    addressesEqual: Object.is
+  }
+  const Aragon = proxyquire.noCallThru().load('./index', {
+    '@aragon/messenger': messengerConstructorStub,
+    './utils': utilsStub
+  }).default
+
+  t.context = {
```

Comment:
Moving all of this into context means they get recreated on each test, so the stubs don't share any history across tests :).